docs/concepts/dotnet-standard-library.md

```diff
@@ -0,0 +1,151 @@
+
+You can see the set of [PCL profiles that are compatible with the .NET Standard](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#portable-profiles), per version.
+
+The PCL compatibility layer is defined:
+
+- [NuGet packages](https://www.nuget.org/packages/Microsoft.NETCore.Compatibility)
+- Other places?
+
+Standard Versions
+-----------------
+
+The .NET Standard is not specific to any one .NET runtime, nor does it match the versioning scheme of any of those runtimes. APIs added to any of the runtimes (such as, Mono, .NET Framework, and .NET Core) can be considered as candidates to add to the specification. New [versions of the .NET Standard Library](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#list-of-net-corefx-apis-and-their-associated-net-platform-standard-version) are created for one of two reasons:
+
+- One or more new APIs are added to a .NET runtime (for example, .NET Core) that are considered fundamental. Other .NET runtimes are required to implement this new API in order to satisfy the new standard version.
+- One or more APIs are deterimed to be implemented across a quorum of .NET runtimes (for example, .NET Framework, .NET Core and Mono), making these APIs candidates for the standard.
+
+Each standard version defines a set of APIs and inherits the APIs defined by previous versions. Some versions add a large set of APIs and others very few. This is in large part because of the reasons listed above, on the reasons why APIs get added to the standard.
+
+The .NET Standard Library was published as multiple versions at its inception (versions 1.0 through 1.5). The creation of multiple versions enabled .NET Standard and PCLs to overlap. This overlap was created for two reasons:
+
+- Enable NSLs to reference PCLs.
+- Enable PCLs to be packaged as NSLs.
+
+Informative Specification
+-------------------------
+
+The standard defines the APIs that are available, per version. The set of APIs is the normative content of the spec. There is other important related information that is informative, extrinsic to the spec. Key informative information:
+
+- [.NET platforms support](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#mapping-the-net-platform-standard-to-platforms) per .NET Standard Library version.
+
+Targeting .NET Standard Library
+===============================
+
+
+
+
+The standard is specified by a set of [NuGet packages](../core-concepts/packages.md). You can compile libraries with these same NuGet packages in order to build libraries that are portable, across .NET Standard compliant runtimes.
+
+It can be thought of as the next generation of [Portable Class Libraries](https://msdn.microsoft.com/library/gg597391.aspx), although is architecturally different. You can learn more 
```

Comment:
Something missing at end of line here.  Also "although is" --> "although it is"
Is there an explanation here somewhere about how standard is "structurally different" from PCL?